### PR TITLE
Display colored fonticons before custom buttons

### PR DESCRIPTION
**Pull request by skateman**
Pivotal story in the OPS UI: https://www.pivotaltracker.com/story/show/147779325
Depends on: https://github.com/ManageIQ/manageiq-schema/pull/39
Some icons might be not displayed becauseof this: https://github.com/ManageIQ/manageiq-ui-service/issues/850

screenshot from 2017-07-22 11-43-45

### CODE REVIEW

**Comment by serenamarie125** [comment]
@chriskacerguis i think should should have a ux/review tag on it along with a UX review?

**Comment by chriskacerguis** [comment]
@serenamarie125 /agree :)

**Comment by chriskacerguis** [comment]
@serenamarie125 I've added the label, but it looks like this is still WIP.

**Comment by skateman** [comment]
@chriskacerguis I market it as WIP because of #850

**Comment by chriskacerguis** [comment]
@skateman ahhh...thanks for clarifying.

**Comment by serenamarie125** [comment]
@chriskacerguis understood, but having visibility to what's being worked on is helpful, thank you!

**Comment by chriskacerguis** [comment]
@serenamarie125 makes sense to me :)

File: client/app/services/custom-button/custom-button.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
           ng-click="vm.invokeCustomAction(button)"
           uib-tooltip="{{button.description}}"
           tooltip-placement="top">
+    <i ng-class="button.options.button_icon" ng-style="{color: button.options.button_color }"></i>&nbsp;
```
Comment (AllenBW) [inline client/app/services/custom-button/custom-button.html]: Skeptical about use of `&nbsp;` why is this necessary?

Comment (skateman) [inline client/app/services/custom-button/custom-button.html]: @AllenBW I can add some padding-right instead

Comment (AllenBW) [inline client/app/services/custom-button/custom-button.html]: yes please!! @skateman 🙇

Comment (chalettu) [inline client/app/services/custom-button/custom-button.html]: So has the API been updated to send the icons in the custom button API response?

**Comment by skateman** [comment]
Rebased, ready to merge!

**Comment by chriskacerguis** [comment]
@serenamarie125 ready for review :)

**Comment by serenamarie125** [comment]
What am i supposed to be reviewing here :)  I haven't seen any icon picker, is that it?  I see one image with a green star, but 👀 unsure

**Comment by skateman** [comment]
@serenamarie125 that's the thing :wink: we're just displaying the icons in the SUI and the [picker](https://github.com/ManageIQ/ui-components/pull/78) will be used in the OPS UI only.

**Comment by chriskacerguis** [comment]
@serenamarie125 echoing the same thing that @skateman said, since this is just the SUI...it is just showing the icons...not picking them.  I think that the image is showing is what the button with an icon / color will look like in the SUI.
